### ISSUE TITLE
Smarter default entity selection 

### ISSUE THREAD

**Issue by mispy** [owid/owid-grapher#71]
Max writes:

– It'd be good if only countries would be selected and not world regions in any definition (Only entities that have an OWID country name should be displayed but not 'middle-income countries' or 'highly indebted low income countries' or similar specific categories.) One exception to this would be that 'World' should be displayed when chosen at random. And the second exception would be that world regions should be shown if the dataset does not contain data on countries and only on world regions; More generally if no country data is available in a dataset than the displayed entities are completely random, as right now.)

– It might be nice to show 5 countries by default. One (at random) from each world region: Europe, Africa, Asia, Oceania, Americas.

– We could think about whether it makes sense to only show countries with a population of over 1 or 5 million.

---

In other words, when creating a new chart:
- Select 5 default keys to show, prioritizing World if available, then high-population countries from different world regions (pregenerate/hardcode a list of candidates), then non-country entities
